### PR TITLE
tauon: 7.6.4 -> 7.6.5

### PR DESCRIPTION
**Pull request by jansol**
###### Description of changes

Not listed but this also finally fixes the occasional loudness surprise when fading between tracks while replaygain is active.
https://github.com/Taiko2k/TauonMusicBox/releases/tag/v7.6.5

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jansol** [comment]
Might as well pull in #227728 while at it.

File: pkgs/applications/audio/tauon/default.nix [hunk 1]
```diff
@@ -25,13 +25,13 @@
 
 stdenv.mkDerivation rec {
```
Comment (drupol) [inline pkgs/applications/audio/tauon/default.nix:26]: Here, I recommend using the `finalAttrs` pattern. To learn more about it, I suggest referring to the following resources:

1. Historical Discussion: The discourse thread at https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293 provides insights into the benefits of avoiding recursive expressions in Nixpkgs.
2. Documentation: The Nixpkgs manual provides detailed information on `mkDerivation` and recursive attributes. You can find it at https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes.
3. Implementation Example: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe.

I hope these resources provide valuable insights and guidance for your needs. If you have any further questions, feel free to ask.

**Review by drupol — COMMENTED** [review]
Hi!

I added a non-blocking comment, feel free to implement it if you want, if not, just let me know and I'll merge this immediately.

**Review by drupol — APPROVED** [review]
Thank you !

**Comment by jansol** [comment]
@drupol can you merge the linked PR as well?

**Pull request merged**